Dockerfile

```diff
@@ -73,13 +74,13 @@ RUN curl -sL https://github.com/postmodern/chruby/archive/v0.3.9.tar.gz | tar xz
     rm -rf /chruby-* && rm -rf /usr/local/src/* && yum clean all
 
 ## GIT clone manageiq-appliance and self-service UI repo (SSUI)
-RUN git clone --depth 1 https://github.com/ManageIQ/manageiq-appliance.git ${APPLIANCE_ROOT} && \
-git clone --depth 1 https://github.com/ManageIQ/manageiq-ui-self_service.git ${SSUI_ROOT} && \
+RUN git clone --depth 1 --branch ${REF} https://github.com/ManageIQ/manageiq-appliance.git ${APPLIANCE_ROOT} && \
+git clone --depth 1 --branch ${REF} https://github.com/ManageIQ/manageiq-ui-self_service.git ${SSUI_ROOT} && \
 ln -vs ${APP_ROOT} /opt/manageiq/manageiq
 
 ## Create approot, ADD miq
 RUN mkdir -p ${APP_ROOT}
-ADD . ${APP_ROOT}
+RUN git clone --depth 1 --branch ${REF} https://github.com/ManageIQ/manageiq.git ${APP_ROOT}
```

Comment:
I think this will solve an issue I noticed today, where About page wasn't showing git sha (.git directory is being excluded from ADD).  If we're not doing ADD, having .git in .dockerignore is no longer necessary?